src/utils/copyText.ts

```diff
@@ -14,6 +14,20 @@ import getEncodedAddress from './getEncodedAddress';
  * @param network network to encode the address string for
  *
  */
+const showMessage = (): void => {
+	message.config({
```

Comment:
This code is not working properly don't use 550 hardcoded use `top: window.innerHeight - 100`

![image](https://github.com/polkasafe/polkasafe-ui/assets/65517966/e29362da-b839-448b-9a80-63b8b1e3fb0e)